test_ws_client.py

```
@@ -0,0 +1,217 @@
+        9: 'Explosion',
+        10: 'Recon Drones'
+    }
+    active_alerts = []
+    for bit, name in alert_names.items():
+        if flags16 & (1 << bit):
+            active_alerts.append(name)
+    return active_alerts if active_alerts else ['NO_ALERT']
+
+
+
+async def send_command(websocket):
```

Ruff (0.14.14)

[warning] Unused function argument: `websocket`

(ARG001)